### PR TITLE
SEO-173239 WPF Chart Axis

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @MallikaRK ,

Have renamed image folder, changed image name and image alt.

Regards,
Yvone.

### CODE REVIEW

**Review by MallikaRK — APPROVED** [review]
Approved

File: wpf/Charts/Axis.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ chart.SecondaryAxis = new NumericalAxis() { Header = "Values(In Tonnes)" };
 
 {% endtabs %}
 
-![WPF ChartAxis with Header](Axis_images/wpf-chart-axis-header.jpeg)
```
Comment (Girija-Rajendran) [inline wpf/Charts/Axis.md:117]: image

Comment (Yvone-Atieno) [reply wpf/Charts/Axis.md:117]: the folder is changed but it has not reflected yet

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**